### ISSUE TITLE
Error in header shown in source file

### ISSUE THREAD

**Issue by steronydh** [atom-community/linter#1098]
I've got a CPP file which includes an H file. linter cannot find the header included in the H file at line 3. In Atom, it shows an error against line 3 of the CPP file against a *different* #include:

Header.h file:

```
#pragma once

#include <missing_header.h>  <<<<<< No error shown here
...
```

CPP file:

```
#include "Header.h"

#include <AnotherHeader.h> <<<< Error shown here
...
```

**Comment by steelbrain** [comment]
This package is actually the base package that provides API to other packages to lint your files. Please reopen this in your linter provider :)

**Comment by steronydh** [comment]
OK! :)
